experiments/formulas/math_mathif_formulas _sumif_avgif_etc.js

```diff
@@ -0,0 +1,163 @@
+// This was written for Skuid 11.X, and has not been tested in other version
```

Comment:
@skuidMatt Definitely like having all of these in one file, but can you rename it to be **camelCase**? Maybe just mathFormulas.js` ?

Reply:
Though to allow room for future submissions, maybe something more specific would be better. Maybe instead break it up with hyphens, which was our secondary naming convention beyond camelCase:

`math-mathif-sumif-avgif-etcFormulas.js`

Reply:
Sounds good to me, if we have a naming convention we want to follow I can change it to that.